Sources/SwifterSwift/UIKit/UIViewExtensions.swift

```diff
@@ -751,4 +751,32 @@ public extension UIView {
     }
 }
 
+
+
```

Comment:
Please take note of all the extra empty lines you're adding. Look at the rest of the file/s and make sure that you're aligning with the project style.